### PR TITLE
lightning: add a limitation during data migration

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add a limitation that users do not perform read/writer operations during data migration. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 1]
```diff
@@ -64,7 +64,9 @@ Physical Import Mode 对应的后端模式为 `local`。
 
 - 使用多个 TiDB Lightning 向同一目标导入时，请勿混用不同的 backend，即不可同时使用 Physical Import Mode 和 Logical Import Mode 导入同一 TiDB 集群。
 
-- 单个 Lightning 进程导入单表不应超过 10TB，使用并行导入 Lightning 实例不应超过 10 个。
+- 在导入数据的过程中，请勿在目标表进行读写操作，否则会导致导入失败或数据不一致。请在导入操作完成后再进行读写操作。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:67]: 在导入数据的过程中，请勿在目标表进行写操作，否则会导致导入失败或数据不一致，期间也不要进行读操作，可能会导致读取的数据不一致。请在导入操作完成后再进行读写操作。

Comment (hfxsd) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:67]:
```suggestion
- 在导入数据的过程中，请勿在目标表进行写操作，否则会导致导入失败或数据不一致。期间也不建议进行读操作，因为读取的数据可能不一致。请在导入操作完成后再进行读写操作。
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 2]
```diff
@@ -64,7 +64,9 @@ Physical Import Mode 对应的后端模式为 `local`。
 
 - 使用多个 TiDB Lightning 向同一目标导入时，请勿混用不同的 backend，即不可同时使用 Physical Import Mode 和 Logical Import Mode 导入同一 TiDB 集群。
 
-- 单个 Lightning 进程导入单表不应超过 10TB，使用并行导入 Lightning 实例不应超过 10 个。
+- 在导入数据的过程中，请勿在目标表进行写操作，否则会导致导入失败或数据不一致。期间也不建议进行读操作，因为读取的数据可能不一致。请在导入操作完成后再进行读写操作。
+
+- 单个 Lightning 进程导入单表不应超过 10 TB，使用并行导入 Lightning 实例不应超过 10 个。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:69]:
```suggestion
- 单个 Lightning 进程导入单表不应超过 10 TB。使用并行导入时，Lightning 实例不应超过 10 个。
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 3]
```diff
@@ -64,7 +64,9 @@ Physical Import Mode 对应的后端模式为 `local`。
 
 - 使用多个 TiDB Lightning 向同一目标导入时，请勿混用不同的 backend，即不可同时使用 Physical Import Mode 和 Logical Import Mode 导入同一 TiDB 集群。
 
-- 单个 Lightning 进程导入单表不应超过 10TB，使用并行导入 Lightning 实例不应超过 10 个。
+- 在导入数据的过程中，请勿在目标表进行写操作，否则会导致导入失败或数据不一致。期间也不建议进行读操作，因为读取的数据可能不一致。请在导入操作完成后再进行读写操作。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:67]:
```suggestion
- 在导入数据的过程中，请勿在目标表进行写操作，否则会导致导入失败或数据不一致。导入期间也不建议进行读操作，因为读取的数据可能不一致。请在导入操作完成后再进行读写操作。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**